doc/API-Friendica.md

```diff
@@ -24,6 +24,28 @@ Returns a list of [Event](help/API-Entities#Event) entities for the current logg
 - `since_id`: (optional) minimum event id for pagination
 - `count`: maximum number of items returned, default 20
 
+### POST api/friendica/event_create
+
+Create a new event for the current logged in user.
+
+#### Parameters
+
+- `id` : (optional) id of event, event will be amended if supplied
+- `name` : name of the event (required)
+- `startTime` : start of the event (ISO), required
```

Comment:
I guess the parameters have to be adjusted to the changes?

Reply:
The API entities documentation was also obsolete, listing `start` and `finish` for the event entity.